src/test/ui/suggestions/issue-82361.stderr

```diff
@@ -0,0 +1,33 @@
+error[E0308]: `if` and `else` have incompatible types
+  --> $DIR/issue-82361.rs:8:9
+   |
+LL | /     if true {
+LL | |         a
+   | |         - expected because of this
+LL | |     } else {
+LL | |         b
+   | |         ^
+   | |         |
+   | |         expected `usize`, found `&usize`
+   | |         help: consider dereferencing the borrow: `*b`
+LL | |     };
+   | |_____- `if` and `else` have incompatible types
+
+error[E0308]: `if` and `else` have incompatible types
+  --> $DIR/issue-82361.rs:14:9
+   |
+LL | /     if true {
+LL | |         1
+   | |         - expected because of this
+LL | |     } else {
+LL | |         &1
+   | |         ^-
+   | |         ||
+   | |         |help: consider removing the `&`: `1`
```

Comment:
The span for the suggestion seems to be wrong.

Reply:
Should be fixed now